app/src/services/api.js

```diff
@@ -0,0 +1,25 @@
+export default {
+
+  getArticles() {
+
+    return fetch('/api/articles_table')
```

Comment:
```suggestion
    return fetch('/api/articles')
```
Best practice for api path names is to follow this pattern: /api/<resources>. I would change all instances of this path moving forward.